tools/sof-dump-status.py

```diff
@@ -217,6 +252,23 @@ def loadPower(self):
         self.sys_power['wakeup_count']=output
         self.sys_power['run_status'] = []
 
+        # check to see if we're on a DT-based platform
+        self.loadDT()
+
+        if self.dt_info:
+            # check to see if we're on i.MX
+            if self.dt_info["soc_id"] in self.imx_soc_id_to_dsp_name:
+                dsp_name = self.imx_soc_id_to_dsp_name[self.dt_info["soc_id"]]
+            else:
+                raise NoMapException("Current DT-based platform doesn't have a soc_id to dsp_name map.")
```

Comment:
This error handling is overkill. This is a test script, a stack trace is enough. In fact I would much prefer a stack trace, gets me right into the code and points me straight to where the problem is.